[perf] Dotnet restore unable to find 8.0.0-rc.* packages

## Description

The perf jobs failed to restore `MicroBenchmarks.csproj`.

```
+ python3 /home/helixbot/work/AAE30989/w/C7D30A73/e/performance/scripts/benchmarks_ci.py --csproj /home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj --incremental no --architecture x64 -f net8.0 --upload-to-perflab-container --dotnet-versions 8.0.100-rc.1.23375.8 --cli-source-info args --cli-branch refs/heads/main --cli-commit-sha 764f77412ef4ec1a7e5dc521608247487a66cd7e --cli-repository https://github.com/dotnet/runtime --cli-source-timestamp 2023-07-25T19:04:35Z --bdn-artifacts /home/helixbot/work/AAE30989/w/C7D30A73/e/artifacts/BenchmarkDotNet.Artifacts --bdn-arguments=--anyCategories Libraries Runtime  --category-exclusion-filter NoAOT NoWASM --runtimes monoaotllvm --aotcompilerpath /home/helixbot/work/AAE30989/p/monoaot/sgen/mini/mono-sgen --customruntimepack /home/helixbot/work/AAE30989/p/monoaot/pack --aotcompilermode llvm --logBuildOutput --generateBinLog  --partition-count 30 --partition-index 1
----------------------------------------------
Initializing logger 2023-07-25 17:20:58.395098
----------------------------------------------
Installing tools.
----------------------
Downloading DotNet Cli
----------------------
DotNet Install Path: '/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/tools/dotnet/x64'
Downloading https://dot.net/v1/dotnet-install.sh
Downloaded https://dot.net/v1/dotnet-install.sh OK
$ pushd "/home/helixbot/work/AAE30989/w/C7D30A73/e/performance"
$ /home/helixbot/work/AAE30989/w/C7D30A73/e/performance/tools/dotnet/x64/dotnet-install.sh -InstallDir /home/helixbot/work/AAE30989/w/C7D30A73/e/performance/tools/dotnet/x64 -Architecture x64 -Version 8.0.100-rc.1.23375.8
dotnet-install: Attempting to download using primary link https://dotnetcli.azureedge.net/dotnet/Sdk/8.0.100-rc.1.23375.8/dotnet-sdk-8.0.100-rc.1.23375.8-linux-x64.tar.gz
curl: (22) The requested URL returned error: 404
dotnet-install: The resource at primary link 'https://dotnetcli.azureedge.net/dotnet/Sdk/8.0.100-rc.1.23375.8/dotnet-sdk-8.0.100-rc.1.23375.8-linux-x64.tar.gz' is not available.
dotnet-install: Attempting to download using legacy link https://dotnetcli.azureedge.net/dotnet/Sdk/8.0.100-rc.1.23375.8/dotnet-dev-ubuntu.18.04-x64.8.0.100-rc.1.23375.8.tar.gz
curl: (22) The requested URL returned error: 404
dotnet-install: The resource at legacy link 'https://dotnetcli.azureedge.net/dotnet/Sdk/8.0.100-rc.1.23375.8/dotnet-dev-ubuntu.18.04-x64.8.0.100-rc.1.23375.8.tar.gz' is not available.
dotnet-install: Attempting to download using primary link https://dotnetbuilds.azureedge.net/public/Sdk/8.0.100-rc.1.23375.8/dotnet-sdk-8.0.100-rc.1.23375.8-linux-x64.tar.gz
dotnet-install: Extracting zip from https://dotnetbuilds.azureedge.net/public/Sdk/8.0.100-rc.1.23375.8/dotnet-sdk-8.0.100-rc.1.23375.8-linux-x64.tar.gz
dotnet-install: Installed version is 8.0.100-rc.1.23375.8
dotnet-install: Adding to current process PATH: `/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/tools/dotnet/x64`. Note: This change will be visible only when sourcing script.
dotnet-install: Note that the script does not resolve dependencies during installation.
dotnet-install: To check the list of dependencies, go to https://learn.microsoft.com/dotnet/core/install, select your operating system and check the "Dependencies" section.
dotnet-install: Installation finished successfully.
$ popd
$ dotnet --info
.NET SDK:
 Version:   8.0.100-rc.1.23375.8
 Commit:    5b3022c008

Runtime Environment:
 OS Name:     ubuntu
 OS Version:  18.04
 OS Platform: Linux
 RID:         ubuntu.18.04-x64
 Base Path:   /home/helixbot/work/AAE30989/w/C7D30A73/e/performance/tools/dotnet/x64/sdk/8.0.100-rc.1.23375.8/

.NET workloads installed:
There are no installed workloads to display.

Host:
  Version:      8.0.0-rc.1.23375.1
  Architecture: x64
  Commit:       d1fc57ea18
  RID:          linux-x64

.NET SDKs installed:
  8.0.100-rc.1.23375.8 [/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/tools/dotnet/x64/sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 8.0.0-rc.1.23374.14 [/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/tools/dotnet/x64/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 8.0.0-rc.1.23375.1 [/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/tools/dotnet/x64/shared/Microsoft.NETCore.App]

Other architectures found:
  None

Environment variables:
  DOTNET_ROOT       [/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/tools/dotnet/x64]

global.json file:
  Not found

Learn more:
  https://aka.ms/dotnet/info

Download .NET:
  https://aka.ms/dotnet/download
---------------------------------------
Removing packages, bin and obj folders.
---------------------------------------
-------------------------------
Restoring .NET micro benchmarks
-------------------------------
$ pushd "/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro"
$ dotnet restore /home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj --packages /home/helixbot/work/AAE30989/w/C7D30A73/e/performance/artifacts/packages /p:UseSharedCompilation=false /p:BuildInParallel=false /m:1
  Determining projects to restore...
  Restored /home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/tools/Reporting/Reporting/Reporting.csproj (in 2.44 sec).
  Restored 
```
..
```
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/harness/BenchmarkDotNet.Extensions/BenchmarkDotNet.Extensions.csproj (in 2.44 sec).
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102: Unable to find package System.Text.Json with version (>= 8.0.0-rc.1.23375.7)
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 2072 version(s) in dotnet6 [ Nearest version: 6.0.2-servicing.1.22101.5 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 1560 version(s) in dotnet7 [ Nearest version: 7.0.0-rtm.22511.4 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 1290 version(s) in dotnet8 [ Nearest version: 8.0.0-rc.1.23375.4 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 107 version(s) in dotnet-public [ Nearest version: 8.0.0-preview.6.23329.7 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 45 version(s) in dotnet3.1 [ Nearest version: 4.7.0-preview3.19561.8 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 1 version(s) in dotnet-eng [ Nearest version: 5.0.0-alpha.1.19618.1 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 0 version(s) in benchmark-dotnet-prerelease
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 0 version(s) in dotnet-tools
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 0 version(s) in dotnet3.1-transport
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102: Unable to find package System.Security.Cryptography.Xml with version (>= 8.0.0-rc.1.23375.7)
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 2069 version(s) in dotnet6 [ Nearest version: 6.0.2-mauipre.1.22102.15 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 1560 version(s) in dotnet7 [ Nearest version: 7.0.0-rtm.22511.4 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 1290 version(s) in dotnet8 [ Nearest version: 8.0.0-rc.1.23375.4 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 69 version(s) in dotnet-public [ Nearest version: 8.0.0-preview.6.23329.7 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 45 version(s) in dotnet3.1 [ Nearest version: 4.7.0-preview3.19561.8 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 1 version(s) in dotnet-eng [ Nearest version: 5.0.0-alpha.1.19618.1 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 0 version(s) in benchmark-dotnet-prerelease
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 0 version(s) in dotnet-tools
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 0 version(s) in dotnet3.1-transport
  Failed to restore /home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj (in 2.82 sec).
$ popd
$ pushd "/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro"
$ dotnet restore /home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj --packages /home/helixbot/work/AAE30989/w/C7D30A73/e/performance/artifacts/packages /p:UseSharedCompilation=false /p:BuildInParallel=false /m:1
  Determining projects to restore...
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102: Unable to find package System.Text.Json with version (>= 8.0.0-rc.1.23375.7)
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 2072 version(s) in dotnet6 [ Nearest version: 6.0.2-servicing.1.22101.5 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 1560 version(s) in dotnet7 [ Nearest version: 7.0.0-rtm.22511.4 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 1290 version(s) in dotnet8 [ Nearest version: 8.0.0-rc.1.23375.4 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 107 version(s) in dotnet-public [ Nearest version: 8.0.0-preview.6.23329.7 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 45 version(s) in dotnet3.1 [ Nearest version: 4.7.0-preview3.19561.8 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 1 version(s) in dotnet-eng [ Nearest version: 5.0.0-alpha.1.19618.1 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 0 version(s) in benchmark-dotnet-prerelease
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 0 version(s) in dotnet-tools
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 0 version(s) in dotnet3.1-transport
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102: Unable to find package System.Security.Cryptography.Xml with version (>= 8.0.0-rc.1.23375.7)
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 2069 version(s) in dotnet6 [ Nearest version: 6.0.2-mauipre.1.22102.15 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 1560 version(s) in dotnet7 [ Nearest version: 7.0.0-rtm.22511.4 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 1290 version(s) in dotnet8 [ Nearest version: 8.0.0-rc.1.23375.4 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 69 version(s) in dotnet-public [ Nearest version: 8.0.0-preview.6.23329.7 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 45 version(s) in dotnet3.1 [ Nearest version: 4.7.0-preview3.19561.8 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 1 version(s) in dotnet-eng [ Nearest version: 5.0.0-alpha.1.19618.1 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 0 version(s) in benchmark-dotnet-prerelease
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 0 version(s) in dotnet-tools
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 0 version(s) in dotnet3.1-transport
  Failed to restore /home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj (in 1.59 sec).
  2 of 3 projects are up-to-date for restore.
$ popd
$ pushd "/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro"
$ dotnet restore /home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj --packages /home/helixbot/work/AAE30989/w/C7D30A73/e/performance/artifacts/packages /p:UseSharedCompilation=false /p:BuildInParallel=false /m:1
  Determining projects to restore...
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102: Unable to find package System.Text.Json with version (>= 8.0.0-rc.1.23375.7)
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 2072 version(s) in dotnet6 [ Nearest version: 6.0.2-servicing.1.22101.5 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 1560 version(s) in dotnet7 [ Nearest version: 7.0.0-rtm.22511.4 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 1290 version(s) in dotnet8 [ Nearest version: 8.0.0-rc.1.23375.4 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 107 version(s) in dotnet-public [ Nearest version: 8.0.0-preview.6.23329.7 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 45 version(s) in dotnet3.1 [ Nearest version: 4.7.0-preview3.19561.8 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 1 version(s) in dotnet-eng [ Nearest version: 5.0.0-alpha.1.19618.1 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 0 version(s) in benchmark-dotnet-prerelease
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 0 version(s) in dotnet-tools
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 0 version(s) in dotnet3.1-transport
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102: Unable to find package System.Security.Cryptography.Xml with version (>= 8.0.0-rc.1.23375.7)
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 2069 version(s) in dotnet6 [ Nearest version: 6.0.2-mauipre.1.22102.15 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 1560 version(s) in dotnet7 [ Nearest version: 7.0.0-rtm.22511.4 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 1290 version(s) in dotnet8 [ Nearest version: 8.0.0-rc.1.23375.4 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 69 version(s) in dotnet-public [ Nearest version: 8.0.0-preview.6.23329.7 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 45 version(s) in dotnet3.1 [ Nearest version: 4.7.0-preview3.19561.8 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 1 version(s) in dotnet-eng [ Nearest version: 5.0.0-alpha.1.19618.1 ]
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 0 version(s) in benchmark-dotnet-prerelease
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 0 version(s) in dotnet-tools
/home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj : error NU1102:   - Found 0 version(s) in dotnet3.1-transport
  Failed to restore /home/helixbot/work/AAE30989/w/C7D30A73/e/performance/src/benchmarks/micro/MicroBenchmarks.csproj (in 1.71 sec).
  2 of 3 projects are up-to-date for restore.
$ popd
 Process exited with status 1
```

/cc: @LoopedBard3 @cincuranet @radical @LeVladIonescu @SamMonoRT

Could somebody from @dotnet/nuget-team please take a look at this? The issue seems to be around at least since ~Jan26 (https://github.com/dotnet/runtime/issues/81222). Using a version string with wildcards causes the restore to race sometimes causing:
```
src/benchmarks/micro/MicroBenchmarks.csproj :   
error NU1102: Unable to find package System.Text.Json with version (>= 8.0.0-rc.1.23375.7)
error NU1102:   - Found 1290 version(s) in dotnet8 [ Nearest version: 8.0.0-rc.1.23375.4 ]
```

The version it is complaining about `8.0.0-rc.1.23375.7` is slightly newer than the latest version (`8.0.0-rc.1.23375.4`) it reports from the `dotnet8` feed.

--- a/src/benchmarks/micro/MicroBenchmarks.csproj
+++ b/src/benchmarks/micro/MicroBenchmarks.csproj
@@ -20,6 +20,7 @@
     <!-- Allow building with one major version, and running using a sdk with a higher major version -->
     <RollForward Condition="'$(BuildingForWasm)' == 'true'">LatestMajor</RollForward>
   </PropertyGroup>
+
   <!-- out-of-band packages that are not included in NetCoreApp have TFM-specific versions -->
   <Choose>
     <When Condition="'$(TargetFramework)' == 'net6.0'">
@@ -39,8 +40,8 @@
     <When Condition="'$(TargetFramework)' == 'net8.0'">
         <PropertyGroup>
             <LangVersion>preview</LangVersion>
-            <ExtensionsVersion>8.0.*-*</ExtensionsVersion>
-            <SystemVersion>8.0.*-*</SystemVersion>
+            <ExtensionsVersion>$(MicrosoftExtensionsLoggingPackageVersion)</ExtensionsVersion>
+            <SystemVersion>$(SystemThreadingChannelsPackageVersion)</SystemVersion>
         </PropertyGroup>
     </When>
     <Otherwise>

